src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -133,5 +133,19 @@ public static void showStudentAtIndex(Model model, Index targetIndex) {
 
         assertEquals(1, model.getFilteredStudentList().size());
     }
+    /**
+     * Updates {@code model}'s filtered list to show only the student at the given {@code targetIndex} in the
+     * {@code model}'s address book.
+     */
+    public static void showSessionAtIndex(Model model, Index targetIndex) {
+        //TODO: List_Session Test
+        assertTrue(targetIndex.getZeroBased() < model.getFilteredStudentList().size());
+
+        Student student = model.getFilteredStudentList().get(targetIndex.getZeroBased());
+        final String[] splitName = student.getName().fullName.split("\\s+");
+        model.updateFilteredStudentList(new NameContainsKeywordsPredicate(Arrays.asList(splitName[0])));
+
+        assertEquals(1, model.getFilteredStudentList().size());
+    }
```

Comment:
Is this method needed?
I think we can use the showStudentAtIndex method for our session command test cases also?

Reply:
I would think so, so that we know the two panels are in sync and the first session will always be from the first student in the list.

Reply:
Ok! 😄